.github/workflows/release.yml

```diff
@@ -84,20 +86,24 @@ jobs:
               generate_release_notes: true
             })
 
-      - name: Check commit messages for feat or break
+      - name: Check commit messages and descriptions for feat or break
         id: check_commits
         run: |
           PUBLISH_DOCS=false
+          COMMIT_DETAILS=""
           for COMMIT in $(git rev-list ${{ github.event.before }}..${{ github.sha }}); do
             COMMIT_MESSAGE=$(git log -1 --pretty=%B $COMMIT)
+            COMMIT_DESCRIPTION=$(git log -1 --pretty=%b $COMMIT)
             echo "Commit message: $COMMIT_MESSAGE"
+            echo "Commit description: $COMMIT_DESCRIPTION"
```

Comment:
When appending commit details to the COMMIT_DETAILS variable, use the correct syntax for newline character. The correct format is `COMMIT_DETAILS="$COMMIT_DETAILS\n- $COMMIT_MESSAGE: $COMMIT_DESCRIPTION"`.